Lekce8HW/Car.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Drawing;
+
+public class Car : Vehicle , IDriveable
+{
+    public string Type { get; }
+	public Car(int id, int type)
+	{
+        ID = id;
+        switch (type) 
+        {
+            case 0:
```

Comment:
fyi - Vim, ze tady je na to jeste brzy (probirali jsme to v pozdejsich lekcich), ale tohle by byl treba krasny kandidat, kde by se namisto stringu pouzil enum s temito hodnotami :)